stock_picking_package_info/__openerp__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2015 AvanzOsc
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Picking Package Info",
+    "version": "9.0.1.0.0",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "author": "OdooMRP team, "
+              "AvanzOSC, "
+              "Tecnativa - Pedro M. Baeza, "
```

Comment:
Of course, Ana. I'm talking of replacing `Tecnativa - Pedro M. Baeza` by only `Tecnativa`. No need to be picky about this.